src/TheOpenNetwork/Address.cpp

```diff
@@ -83,12 +83,16 @@ Address::Address(const std::string& string) {
         Data decoded = decodeUserFriendlyAddress(string);
         addressData.workchainId = decoded[1];
 
-        byte tag = decoded[0];
-        if (tag & AddressTag::TEST_ONLY) {
-            isTestOnly = true;
-            tag ^= AddressTag::TEST_ONLY;
+        if (!bounceable.has_value()) {
```

Comment:
optional: at first check positive condition

Reply:
Yes either way work, anyway we will move everything to rust - same result